ci/cppcheck.sh

```diff
@@ -36,6 +36,9 @@ build_cppheck() {
 	wget http://mirrors.kernel.org/ubuntu/pool/universe/t/tinyxml2/libtinyxml2-6a_7.0.0+dfsg-1build1_amd64.deb
 	sudo dpkg -i ./libtinyxml2-6a_7.0.0+dfsg-1build1_amd64.deb
 
+	wget https://launchpad.net/~ubuntu-security-proposed/+archive/ubuntu/ppa/+build/23754764/+files/libpcre3_8.39-9ubuntu0.1_amd64.deb
```

Comment:
`http://launchpadlibrarian.net/601696753/libpcre3_8.39-9ubuntu0.1_arm64.deb` easier to read. same for the rest.